tpm2tools/keys.go

```diff
@@ -49,6 +49,18 @@ func EndorsementKeyFromNvIndex(rw io.ReadWriter, idx uint32) (*Key, error) {
 	return KeyFromNvIndex(rw, tpm2.HandleEndorsement, idx)
 }
 
+// Load SRK based on the given tpm2.Algorithm.
+func LoadSRK(rwc io.ReadWriter, algo tpm2.Algorithm) (*Key, error) {
```

Comment:
I think it would be more reasonable to have `getSRK` and `getEK` helper functions in `gotpm/cmd/utils.go`. We can move these to the public API of `tpm2tools` later if that becomes necessary.

Reply:
Done